finalPrep/README.md

```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+ ![Screen Shot 2022-04-20 at 3 27 29 PM](https://user-images.githubusercontent.com/102986835/164326513-944c9e71-9cb7-4f2a-a8df-5a1c3b2e6143.png)
+ ![Screen Shot 2022-04-20 at 3 28 09 PM](https://user-images.githubusercontent.com/102986835/164326863-469a96ee-4bcf-461f-b603-6b945ac063e0.png)
```

Comment:
This looks great - a realistic time commitment to Turing (including shutting it down earlier on Fridays) and some life stuff.